<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::AccessibilityWidget</class>
 <widget class="QWidget" name="GammaRay::AccessibilityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="a11yTreeSearchLine"/>
       </item>
       <item>
        <widget class="QTreeView" name="a11yTree">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="indentation">
          <number>10</number>
         </property>
         <property name="uniformRowHeights">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="GammaRay::PropertyWidget" name="a11yPropertyWidget" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GammaRay::PropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/propertywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
